content/cpp/concepts/loops/loops.md

```diff
@@ -43,3 +44,28 @@ for (int i = 0; i < 10; i++) {
   std::cout << i << "\n";
 }
 ```
+
+## For-Each Loop
+
+A `for-each` loop iterates over every element in a container (array, vector, etc.). 
+
+The example below uses `for-each` loop to print 1 to 5 from a vector `container`:
+
+> **Note:** The variable is a copy of the object in the vector. To act on the object in the vector, declare the variable by reference (&num).
+
+```cpp
+vector<int> container = {1, 2, 3, 4, 5};
+for (int num : container) {
+  std::cout << num << "\n";
+}
```

Comment:
You need to copy and paste the entire code in your file that gives you the following output that you've now added.